packages/integrations/vercel/README.md

```diff
@@ -129,6 +133,27 @@ export default defineConfig({
 });
 ```
 
+### analytics
+
+> **Type:** `boolean`
+> **Available for:** Serverless, Edge
```

Comment:
This is optional, but have you tried it in static mode? Maybe it works by adding those few lines inside `astro:config:setup`.

Reply:
Sure I'll try to add it in static mode